data/versions.yml

```diff
@@ -2,4 +2,35 @@
 main: "1.25"
 
 # The version of Istio currently documented in preliminary.istio.io
-preliminary: "1.26"
+preliminary: "1.27"
+
+supported_versions:
+  - latest
+  - v1.25
+  - v1.24
+  - v1.23
+  - v1.22
+  - v1.21
+  - v1.20
```

Comment:
(a) we shouldn't need to list all these
(b) we only want to search N-5; so for 1.26, that would be 1.21 and up